rnd/autogpt_builder/src/components/Flow.tsx

```diff
@@ -13,6 +13,9 @@ import ReactFlow, {
   Connection,
 } from 'reactflow';
 import 'reactflow/dist/style.css';
+import { ChevronLeftIcon, ChevronRightIcon, PlayIcon } from '@radix-ui/react-icons';
+import { Button } from './ui/button';
```

Comment:
I *think* the flow changes here might conflict with the work Bently is doing rn

Reply:
I thought I reverted those changes... standby

Reply:
fixed